### DISCUSSION TITLE
Psycopg missing error again but on ubuntu 22.04

### DISCUSSION THREAD

**Discussion by Rammurthy5** [pgEdge/cli#15] [Q&A]
It was working until last day but suddenly its throwing the same psycopg error over and over again :/ 
Using ubuntu 22.04 EC2 vm.
faced the error when i execute this `./nodectl spock node-create n1 'host=<privateIP of node1> user=pgedge dbname=demo' demo`
I dont know what has changed. can someone pls help?

**Comment by luss** [comment]
you likely have to run:

'./nc install nclibs" if you are now running NodeCtl 23.122 or higher and
have gotten there from a 'nc upgrade' rather than thru a fresh install.

**Reply by Rammurthy5** [reply]
nc upgrade says missing port number. sorry, what is the port number i should provide?

**Comment by luss** [comment]
Can you please run an "./nc info" and paste results here.

**Reply by Rammurthy5** [reply]
```
     NodeCtl: v23.123  /home/ssm-user/pgedge
# User & Host: ssm-user  ip-<privateIP>  <privateIP>
#          OS: Ubuntu 22.04.2 LTS  glibc-2.35-el9
#     Machine: 1 GB, 1 vCPU, Intel Xeon E5-2676 v3 @ 2.40GHz
#  Cloud Info: aws  ec2  i-0c70894bc98f90cb7  t2.micro  eu-west-1a
#       Langs: Python v3.10.6 | Perl v5.34.0
#    Repo URL: https://pgedge-download.s3.amazonaws.com/REPO
# Last Update: 2023-07-21 11:15:28
```

**Comment by luss** [comment]
Ugh, this is my bad!!   In NodeCtl 23.123 (pushed yesterday afternoon to
PROD for a completely different reason) I created a regression that broke
support for Ubuntu 22.04.   I will fix this in a push this afternoon of
NodeCtl 23.124 to PROD.  Please wait until I tell you and then please do an
'./nc update' and it should fix your problem.  Pretty much all of our early
trial customers are using "Rocky Linux 9.0" and it is now apparent that our
regressions tests doesn't yet cover Ubu22.
